.github/workflows/github-actions-build.yml

```diff
@@ -0,0 +1,155 @@
+name: GitHub Actions for pega helm chart
+
+env:
+  HELM_URL: https://get.helm.sh
+  HELM_TGZ: helm-v3.2.4-linux-amd64.tar.gz
+  YAMLLINT_VERSION: 1.15.0
+  GO_VERSION: 1.13.1
+
+
+on: 
+  push:
+    tags:
+      - '*'
+    branches-ignore:
+      - gh-pages
+  pull_request:
+    branches: [ master ]
+
+concurrency:
+  group:  ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
```

Comment:
I recommend splitting the lint steps from the Go test steps so they can run in parallel and not block on another by failures. You will need to name the jobs and use dependencies to ensure they all run, but we could then organize it like this: https://github.com/dcasavant/pega-helm-charts-incubator/pull/1/files#diff-4fe883216c20c1d7d34f343f489f9b5a4ab47a0e254226494e9a95a679e493e4R17

Reply:
By the way - that PR was a work in progress and should not be treated as 'done' :)

Reply:
seperated the lint and the go test jobs, now these will run in parallel, and the deploy job will start sunning once all the others run successfully